frameworks/Go/falcore/src/framework_benchmarks/falcore.go

```diff
@@ -182,13 +182,21 @@ var dbFilter = falcore.NewRequestFilter(func(req *falcore.Request) *http.Respons
 // Test 3: Multiple database queries
 var queriesFilter = falcore.NewRequestFilter(func(req *falcore.Request) *http.Response {
 	if req.HttpRequest.URL.Path == "/queries" {
+
 		n := 1
+	
 		if nStr := req.HttpRequest.URL.Query().Get("queries"); len(nStr) > 0 {
-			n, _ = strconv.Atoi(nStr)
+			n, _ = strconv.Atoi(nStr) // rvalue is 0 if nStr is non-number.
 		}
 
-		if n <= 1 {
-			return dbFilter.FilterRequest(req)
+		// In the case of nStr is non-number string, n will be 0. So need to change n to 1.
+		if n == 0 {
+			n = 1
+		}
+
+		// In the case of nStr is number and its value is higher than 500, change n to 500
+		if n > 500 {
```

Comment:
@msmith-techempower 
Hahaha.. sorry Mike. I blame VirtualBox. Sometimes I am not able to access VM using ssh( I still don't know a reason why. its weird), so I just did my work using vim on the VM console without text editor. Let me fix this problem.